source/guides/azure/infiniband.md

```diff
@@ -257,10 +257,9 @@ Then start a new shell.
 
 Create a conda environment (see [UCX-Py](https://ucx-py.readthedocs.io/en/latest/install.html) docs)
 
-```shell
+````shell
```

Comment:
```suggestion
```shell
```

There should only be 3 backticks here.

Reply:
Hmmm this is weird - I vividly remember making sure all the chunks had 3 backticks - seems like every time I manually run prettier or black some miscellaneous backticks get added in. Is there any way to avoid that?

Reply:
If one of those tools added back in a 4th backtick, then something else in the file had formatting errors that confused it.

I recommend generally getting in the habit of self-reviewing the diff for your PRs, like:

* run `git diff` locally before committing changes
* look at the visual diff in https://github.com/rapidsai/deployment/pull/465/files after pushing

> *Is there any way to avoid that?*

Only run those tools like `black` / `prettier` via `pre-commit`. By running the following from the root of the repo.

```shell
pre-commit run --all-files
```

Reply:
This is still incorrect.
```suggestion
```shell
```